injectable.gemspec

```diff
@@ -36,7 +36,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 2.0"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_runtime_dependency 'activesupport', '~> 5.2'
```

Comment:
I'd like this dependency removed. We shouldn't need to use ActiveSupport for this gem.

Reply:
That's the plan, but I want to get this set up as a gem first, then start removing dependencies in separate PRs.

Reply:
I'll have to live with that